src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/Review/index.tsx

```diff
@@ -118,6 +118,10 @@ const ContractInteractionReview = ({ closeSnackbar, enqueueSnackbar, onClose, on
         </Row>
         {tx.selectedMethod.inputs.map(({ name, type }, index) => {
           const key = `methodInput-${tx.selectedMethod.name}_${index}_${type}`
+          let value = tx[key]
+          if (type === 'bool') {
+            value = tx[key] ? String(tx[key]) : 'false'
+          }
```

Comment:
can you move this out to a helper function?

I'm thinking this may be extended in future usages.

Reply:
Sure, good idea, it's done now